src/commands/publish/route.rs

```diff
@@ -28,7 +28,9 @@ impl Route {
 }
 
 fn multi_script(user: &User, script: String) -> Result<Route, failure::Error> {
-    let pattern = &user.settings.clone().project.route.expect("⚠️ Your project config has an error, check your `wrangler.toml`: `route` must be provided.");
+    let pattern = &user.settings.clone().project.route.expect(
+        "⚠️ Your project config has an error, check your `wrangler.toml`: `route` must be provided.",
+    );
```

Comment:
I'm not sure how this change got in here, other than a change to `cargo fmt`?

Reply:
could have been! i'll keep an eye out for that